attestation/collection.go

```diff
@@ -29,27 +30,34 @@ type Collection struct {
 }
 
 type CollectionAttestation struct {
-	Type        string   `json:"type"`
-	Attestation Attestor `json:"attestation"`
+	Type        string    `json:"type"`
+	Attestation Attestor  `json:"attestation"`
+	StartTime   time.Time `json:"start_time"`
+	EndTime     time.Time `json:"end_time"`
```

Comment:
i don't believe we use underscores in any other json tags, so these should probably be `starttime` and `endtime` for consistency

Reply:
thank you